### ISSUE TITLE
Question: is close button for JTabbedPane available?

### ISSUE THREAD

**Issue by Gaarco** [JFormDesigner/FlatLaf#31]
As the title says I'm wondering if there is a function or property to display a close button near the title of a tabbed pane, like the one Intellij uses when you open many files.
It would be nice to have so that we don't have to deal with different LaFs.
Reference:
![githubq](https://user-images.githubusercontent.com/9869590/71643849-11f70b00-2cbf-11ea-907d-75b62c54e550.png)

**Comment by DevCharly** [comment]
No, this is not yet implemented. Maybe in the future...

You could try following:
~~~java
JPanel tab = new JPanel( new BorderLayout( 5, 0 ) );
tab.setOpaque( false );

JButton closeButton = new JButton( new FlatInternalFrameCloseIcon() );
closeButton.setContentAreaFilled( false );
closeButton.setBorder( null );
//TODO add action listener
tab.add( closeButton, BorderLayout.EAST );

tab.add( new JLabel( "Tab Title" ), BorderLayout.CENTER );

tabbedPane1.setTabComponentAt( 0, tab );
~~~

You could also try `JideTabbedPane` from [JIDE Common Layer](https://github.com/jidesoft/jide-oss), which is supported by FlatLaf: https://github.com/JFormDesigner/FlatLaf/tree/master/flatlaf-jide-oss

**Comment by swordmaster2k** [comment]
I came across a similar need, and tried out your suggestion above. It worked very nicely in my case:

![image](https://user-images.githubusercontent.com/3354016/77949260-8d685880-72be-11ea-9f68-3ea6a2d0d5be.png)

The only tweek I made was to set the max size of the button:

```java
JButton closeButton = new JButton(new FlatInternalFrameCloseIcon());
closeButton.setContentAreaFilled(false);
closeButton.addMouseListener(new CloseListener(tab));
closeButton.setPreferredSize(new Dimension(16, 16));
closeButton.setMaximumSize(new Dimension(16, 16));
closeButton.setBorder(null);
```java

Thank you!

**Comment by DevCharly** [comment]
implemented in PR #193
